package.json

```diff
@@ -75,19 +77,16 @@
     "test": "jest",
     "test:db:docker": "printf '\\e[2J\\e[3J\\e[H' && npm run migrate && NODE_ENV=development npm run test",
     "test:restart": "docker-compose -f docker-compose.test.yaml restart tests && docker logs -f pointnetwork_tests",
-    "start": "./point --datadir $DATADIR -v",
-    "start:dev": "nodemon ./point -- --datadir $DATADIR -v",
-    "start:docker": "./scripts/patch-config.js && npm run migrate && npm start",
-    "start:docker:dev": "./scripts/patch-config.js && npm run migrate && npm run start:dev",
-    "start:docker:e2e": "./scripts/patch-config-e2e.js && npm run migrate && npm run start:dev",
-    "migrate": "./point migrate --datadir $DATADIR",
+    "start": "./point",
+    "start:docker": "./point migrate && ./point",
```

Comment:
Let's reuse the scripts here and one line below. Like `"start:docker": "npm run migrate && npm start"`.